src/zope/testbrowser/tests/test_browser.py

```diff
@@ -82,6 +83,45 @@ def __call__(self, environ, start_response):
         return [next_response['body']]
 
 
+class TestDisplayValue(unittest.TestCase):
+    """Testing ..browser.Browser.displayValue."""
+
+    def setUp(self):
+        super(TestDisplayValue, self).setUp()
+        app = TestApp()
+        app.set_next_response(b'''\
+<html>
+  <body>
+    <form>
+      <select name="sel1">
+        <option value="op">Turn</option>
+        <option value="alt">Alternative</option>
+      </select>
+    </form>
+  </body>
+</html>
+''')
```

Comment:
Personally I find tests more readable when they use [textwrap.dedent](https://docs.python.org/2/library/textwrap.html#textwrap.dedent) to have the entire test body indented more than the `def` line.  Also that way my Vim folding doesn't break ;)